tests/test_dagger.py

```diff
@@ -25,62 +29,146 @@ def test_beta_schedule():
 
 
 def test_traj_collector(tmpdir):
-    env = gym.make(ENV_NAME)
+    venv = vec_env.DummyVecEnv([lambda: gym.make(ENV_NAME)])
     robot_calls = 0
 
-    def get_random_act(obs):
+    def get_random_acts(obs):
         nonlocal robot_calls
         robot_calls += 1
-        return env.action_space.sample()
+        return [venv.action_space.sample() for _ in range(venv.num_envs)]
 
     collector = dagger.InteractiveTrajectoryCollector(
-        env=env, get_robot_act=get_random_act, beta=0.5, save_dir=tmpdir
+        venv=venv, get_robot_acts=get_random_acts, beta=0.5, save_dir=tmpdir
     )
     collector.reset()
-    zero_action = np.zeros((), dtype="int")
-    obs, rew, done, info = collector.step(zero_action)
-    assert rew != 0
-    assert not done
-    assert isinstance(info, dict)
+    zero_acts = np.zeros((venv.num_envs,), dtype="int")
+    obs, rews, dones, infos = collector.step(zero_acts)
+    assert rews != 0
```

Comment:
I'm confused how this can work, normally `bool(np.array(...))` throws an error. Should this be `np.all(rews != 0)`?

Reply:
(probably bad port of test from Env to VecEnv)

Reply:
Interesting edge-case here that allowed this to not error:
`array([1.], dtype=float32) != 0` => `array([True])`, and `bool(array[True]) is True`.